packages/delivery-electron/delivery.js

```diff
@@ -16,7 +16,10 @@ const delivery = (client, filestore, net, app) => {
       }
     })
 
-    req.on('error', cb)
+    req.on('error', err => {
+      err.isRetryable = true
```

Comment:
Do we definitely want to retry for every kind of error?

Some errors can happen after the server has responded, so could possibly lead to duplicate events — not sure whether that's a valid concern though, since a successful response really shouldn't have an error 🤔

Reply:
Huh, that's a case I hadn't considered :thinking:

In that case, maybe `isRetryable` should only be set if the response callback hasn't been invoked? Though I'm struggling to come up with a scenario to test it.

Reply:
Yeah that sounds like a good solution, but does seem tricky to test for real. A unit test could probably force it to happen by manuallyemitting an error event after returning a response from the mock server

Reply:
The mock setup in the unit tests seems to perceive an emitted error as an uncaught exception (and automatically fail the test) for some reason. I'll revisit this.